README.md

```diff
@@ -2,9 +2,16 @@
 This is an OpenShift plugin to ElasticSearch to:
 
 * Dynamically update the SearchGuard ACL based on a user's name
-* Transform kibana index requests to support multitenant deployments
+* Transform kibana index requests to support multitenant deployments for non-operations users.
 * Seed the Searchguard index `config`, `roles`, `rolesmapping`, and `actiongroups` types
 
+*Note:*
+Previous versions of this plugin created a Kibana profile for each user regardless of their role.
+As of v2.4.4.2, any users that has been given the OpenShift cluster-admin role will share the same
+Kibana index to store configurations and dashboards.  This change makes it possible to share dashboards
+between operation users.  It is highly recommended that operations teams establish agreements and
+naming conventions so users do not overwrite each others work.
```

Comment:
So we need to consider this change a bit in terms of how one would migrate to start using it.

What happens if we deploy this where each operation user has built up a set of dashboards already?  What happens to access to those previously created dashboards?

It would be great to have an "opt-in" switch to turn it on, so that admins can prepare by first exporting the dashboards they want.

Even with the opt-in switch it would be great to be able to provide a tool to dump each ops user's individual kibana dashboard so that they could be imported to the common one.